locations/spiders/shell.py

```diff
@@ -12,14 +22,28 @@ def parse_item(self, item, location):
         amenities = location["amenities"]
         fuels = location["fuels"]
         apply_category(Categories.FUEL_STATION, item)
-        if "shop" in amenities:
+        if "twenty_four_hour" in amenities:
+            item["opening_hours"] = OpeningHours()
+            item["opening_hours"].add_days_range(DAYS, "00:00", "23:59")
+        if "shop" in amenities or "selectshop" in amenities:
             apply_category(Categories.SHOP_CONVENIENCE, item)
-        apply_yes_no("amenity:chargingstation", item, "electric_charging_other" in fuels or "shell_recharge" in fuels)
-        apply_yes_no("toilets", item, "toilet" in amenities)
-        apply_yes_no("atm", item, "atm" in amenities)
-        apply_yes_no("car_wash", item, "carwash" in amenities)
-        apply_yes_no("hgv", item, "hgv_lane" in amenities)
-        apply_yes_no("wheelchair", item, "disabled_facilities" in amenities)
+        if "restaurant" in amenities:
+            apply_category(Categories.RESTAURANT, item)
```

Comment:
`food=yes` is an OSM attribute for saying that a place also serves food. It's typically used on pubs, but I think it applies here.

Reply:
For a roadhouse/roadside service centre with an apron for refueling cars and building that contains a convenience store and separate restaurant area with seating, does this become 3 POIs? And air compressors, car washes, etc become separate POIs too?

Typically we'll only get one coordinate to share for all POIs, but an ATP consumer can see that there are different related POIs all in the same nearby vicinity.

Reply:
From an OSM perspective, yes, a fuel station will eventually get "micro mapped" into several POIs. [eg](https://www.openstreetmap.org/node/9101318090#map=19/-37.75909/144.97475&layers=D)

From an ATP perspective, we don't add any value when splitting one `amenity=fuel` + `compressed_air=yes` into 2 POIs (`amenity=fuel` and `amenity=compressed_air`). But we do add value when we have specific details about them, like different "brands", if we know there is a Shell fuel station, a Budgens convenience store and a Subway takeaway we should output that. [eg](https://www.openstreetmap.org/way/395120804#map=19/51.36058/-1.14869&layers=D)